sklearn/utils/estimator_checks.py

```diff
@@ -204,7 +205,7 @@ def set_fast_parameters(estimator):
             and estimator.__class__.__name__ != "TSNE"):
         estimator.set_params(n_iter=5)
     if "max_iter" in params:
-        # NMF
+        warnings.simplefilter("ignore", ConvergenceWarning)
```

Comment:
hum, max_iter is also in many other estimators, right? Not sure why I put NMF here then... well, never mind